src/lib.rs

```diff
@@ -185,6 +187,28 @@ impl<T> Glog<T> {
         }
     }
 
+    /// Sets whether or not the span name is included. Defaults to true.
+    ///
+    /// If span names are not included, then the fields from all spans are
+    /// printed as a single list of fields.  This compacts the output and reduces
+    /// the complexity of the output's structure.
```

Comment:
(You don't need double spaces!)

```suggestion
    /// If span names are not included, then the fields from all spans are
    /// printed as a single list of fields. This results is a more compact output.
```

Reply:
Ah sorry! Habit...